src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/TripleDES/TripleDESCipherOneShotTests.cs

```diff
@@ -557,6 +577,517 @@ public static IEnumerable<object[]> TestCases
+
+                    PaddingMode.ANSIX923,
+                    CipherMode.CFB,
+                    8,
+                };
+
+                yield return new object[]
+                {
+                    // plaintext
+                    new byte[]
+                    {
+                        0x99, 0x50, 0x68, 0x12, 0xA4, 0x5F, 0x08, 0xC8,
+                        0x89, 0xB9, 0x7F, 0x59, 0x80, 0x03, 0x8B, 0x83,
+                        0x59,
+                    },
+
+                    // ciphertext
+                    new byte[]
+                    {
+                        0x8F, 0xD6, 0x1C, 0xF3, 0xAF, 0x0D, 0x3C, 0x98,
+                        0xCF, 0x29, 0x20, 0xB3, 0xF3, 0xFC, 0x34, 0xF0,
+                        0x38, 0xA0,
+                    },
+
+                    PaddingMode.ISO10126,
+                    CipherMode.CFB,
+                    8,
+                };
+
+                yield return new object[]
+                {
+                    // plaintext
+                    Array.Empty<byte>(),
+
+                    // ciphertext
+                    new byte[]
+                    {
+                        0x17,
+                    },
+
+                    PaddingMode.PKCS7,
+                    CipherMode.CFB,
+                    8,
+                };
+
+                yield return new object[]
+                {
+                    // plaintext
+                    Array.Empty<byte>(),
+
+                    // ciphertext
+                    Array.Empty<byte>(),
+
+                    PaddingMode.Zeros,
+                    CipherMode.CFB,
+                    8,
+                };
+
+                yield return new object[]
+                {
+                    // plaintext
+                    Array.Empty<byte>(),
+
+                    // ciphertext
+                    Array.Empty<byte>(),
+
+                    PaddingMode.None,
+                    CipherMode.CFB,
+                    8,
+                };
+
+                // 3DES CFB64 is not supported on Windows 7.
+                if (PlatformDetection.IsNotWindows7)
+                {
+                    yield return new object[]
+                    {
+                        // plaintext
+                        new byte[]
+                        {
+                            0x50, 0x68, 0x12, 0xA4, 0x5F, 0x08, 0xC8, 0x89,
+                            0xB9, 0x7F, 0x59, 0x80, 0x03, 0x8B, 0x83, 0x59,
+                        },
+
+                        // ciphertext
+                        new byte[]
+                        {
+                            0x46, 0x82, 0xB5, 0xFC, 0x6A, 0xD3, 0x92, 0xF9,
+                            0xB1, 0xF6, 0xD9, 0xF3, 0xBB, 0x8D, 0x57, 0xE5,
+                            0xFF, 0x66, 0x88, 0x3C, 0x53, 0xC4, 0x5A, 0xC6,
+                        },
+
+                        PaddingMode.PKCS7,
+                        CipherMode.CFB,
+                        64,
+                    };
```

Comment:
The if this is associated with describes that DES3+CFB64 doesn't work on Windows 7, but there's only one test case in the block and 11 that were added unconditionally.

I think this close curly is supposed to move down a fair bit :)

Reply:
I'm not sure... I understand here. The `if` and it's closing brace appear to be in the right place to me... that curly brace you left a comment on is for the `new object[]` construction, not the `if`.

Reply:
So it is.  And everything's properly indented, so it's not that.

Apparently something just mis-fired in my brain.